src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
```

Comment:
According to the checklist, avoid using absolute paths and provide a relative path to resources. The current implementation does not ensure that. Consider refactoring the method to accept a relative path or ensure the provided path is relative.